examples/peer/migrations/2_deploy_contracts.js

```diff
@@ -2,11 +2,13 @@ const Peer = artifacts.require('Peer')
 const Swap = artifacts.require('@airswap/swap/contracts/Swap.sol')
 const Types = artifacts.require('@airswap/types/contracts/Types.sol')
 
+const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
can we use: 
`const { EMPTY_ADDRESS } = require('@airswap/order-utils').constants`

Reply:
Not as easily as we can in tests due to the fact that `web3` is not available in `truffle migrate` (whereas it is available in `truffle test`). Seemed easier to just define the constant in here as opposed to importing web3 purely for the purpose of making the constant imports work.

Reply:
> web3 is not available in truffle migrate
🤦‍♂